pkg/imageserver/webdav.go

```diff
@@ -69,6 +69,9 @@ func (s *webdavImageServer) GetHandler(meta *iiapi.InspectorMetadata,
 		if err := syscall.Chroot(ImageServeURL); err != nil {
 			return nil, fmt.Errorf("Unable to chroot into %s: %v\n", ImageServeURL, err)
 		}
+		if err := syscall.Chdir("/"); err != nil {
```

Comment:
@php-coder pointed me to this bug.

It's generally advised to perform ``chdir(dir)`` first, then perform ``chroot(dir)`` followed by ``seteuid(user)`` to prevent chroot jail breaks. Some resources claim that ``chdir(dir)`` inside the chroot is sufficient, too. I prefer ``chdir(dir)`` first because it separates check for presence of directory from the actual ``chroot()`` call.